on-prem-to-cloud/README.md

```diff
@@ -0,0 +1,61 @@
+# Connectivity: Accessing a Workload in SAP BTP, Kyma Runtime From the Corporate Network
+
+## Context
+
+If you are using a corporate network, you can access a workload located in SAP BTP, Kyma runtime. The access is possible using the SAP BTP, Connectivity. See the following diagram that presents all the involved elements and the high-level flow:
+
+![On Premise to Cloud](./assets/on-prem-to-cloud.drawio.svg)
+
+## Prerequisites
+
+* [curl](https://curl.se/)
```

Comment:
we should add more prerequisites such as 
- kyma runtime
- cloud connector setup in the corporate
- connectivity proxy module enabled in kyma
- access via kubectl enabled to kyma (required tooling rightly configured).

you can check other samples

Reply:
How about:
```suggestion
* SAP BTP, Kyma runtime instance
* [curl](https://curl.se/)
* [Docker](https://www.docker.com/)
* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
* You have the Istio and API Gateway modules added. See [Adding and Deleting a Kyma Module](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module?version=Cloud).
* You have Cloud Connector set up in the corporate network. See [Configure a Service Channel for a Kubernetes Cluster](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/configure-service-channel-for-kubernetes-cluster?version=Cloud).
```